common/templates/deployment.yaml

```diff
@@ -0,0 +1,42 @@
+{{- if .Values.test -}}

```

Comment:
Shame this has to be part of the template but presumably that is just because the common chart doesn't really render anything in isolation. It is more of a helper and to test the helper we need something to test it on.

Reply:
@ryandawsonuk This is correct for common chart because it does not render anything. We can also apply helm unit test plugin to any other chart consuming common chart...